projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1620,7 +1620,7 @@ public function replace_tokens_with_input( $subject, $field_values ) {
 	 * @param array $widget The widget data.
 	 */
 	public function track_current_widget( $widget ) {
-		$this->current_widget_id = $widget['id'];
+		$this->current_widget_id = isset( $widget['id'] ) ? $widget['id'] : '';
```

Comment:
Could also simplify to this:
```suggestion
		$this->current_widget_id = $widget['id'] ?? '';
```